src/TsTypes.ts

```diff
@@ -1,5 +1,9 @@
 import { camelCase, upperFirst } from 'lodash'
 
+const multiLineCommentStart =  '/** '
+const multiLineCommentIndent = '    '
```

Comment:
@bcherny - Done. I put it in same pull request, but two separate commits. I also rebased on top of latest `master` branch. See https://github.com/bcherny/json-schema-to-typescript/pull/26/commits/aef14ff462a106b76eadd0be8babc08e26fec14b